DDDEastAnglia.Tests/Controllers/SessionControllerTests.cs

```diff
@@ -86,7 +87,7 @@ public static SessionController Create(IConference conference)
             {
                 var conferenceLoader = Substitute.For<IConferenceLoader>();
                 conferenceLoader.LoadConference().Returns(conference);
-                var controller = new SessionController(conferenceLoader, Substitute.For<IUserProfileRepository>(), Substitute.For<ISessionRepository>(), Substitute.For<ISessionSorter>());
+                var controller = new SessionController(conferenceLoader, Substitute.For<IUserProfileRepository>(), Substitute.For<ISessionRepository>(), Substitute.For<ISessionSorter>(), Substitute.For<ISessionSubmissionEmailSender>());
```

Comment:
A note for the technical debt/refactoring backlog: it looks as though `ISessionRepository` and `ISessionSorter` are related concerns that can be hidden behind a façade service.

Our dependencies are seem to be getting out of control somewhat.